src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -1547,9 +1514,8 @@ internal bool LoadModuleManifestData(ExternalScriptInfo scriptInfo, ManifestProc
 
             string actualRootModule = moduleToProcess ?? rootModule;
 
-            bool actualRootModuleIsXaml = false;
-            // For workflow modules, the actualRootModule is added to workflowsToProcess and is nulled out.
-            // We need to save this name so that we can assign this to the RootModule property of ModuleInfo
+            // If the root module is a workflow module, the 'actualRootModule' is nulled out.
+            // We need to save this name so that we can assign this to the RootModule property of ModuleInfo.
             string savedActualRootModule = actualRootModule;
```

Comment:
Is this still needed?

Reply:
Yes, I believe it's still needed. In the case that `bailOnFirstError == false`, it will continue to process the module manifest despite the error of the XAML `actualRootModule`. In that case, the `actualRootModule` needs to be null'ed out so that the code enclosed in `if (!string.IsNullOrEmpty(actualRootModule))` and `if (actualRootModule != null)` in the code path below can be skipped as before this change.

Reply:
ok